.github/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -18,205 +18,7 @@ body:
       # NOTE: The list below is autogenerated using `make generate-gh-issue-templates`
       # Do not manually edit it.
       # Start Collector components list
-      - cmd/configschema
```

Comment:
It looks like literal `\n` characters were added here instead of newlines, probably something is wrong on the changes to the scripts. Could you double check?

Reply:
I just ran the command `make generate-gh-issue-templates` locally which the CI was running and it did these changes automatically. That particular job is passing now though. I'll revert changes to this file then.